test/visual-diff/visual-tests

```diff
@@ -94,6 +94,14 @@
       "url": "examples/visual-tests/amp-layout/amp-layout.amp.html",
       "name": "AMP Layout"
     },
+    {
+      "url": "examples/visual-tests/article-fade-in.amp.html",
+      "name": "amp-fx='fade-in' and amp-fx='fade-in-scroll' Visual tests",
```

Comment:
Remove `Visual tests` from the name, as it's redundant. Preferably, use just the component name.
```
 "name": "Fade in",
```